hswidget/cof_widget.go

```diff
@@ -0,0 +1,668 @@
+package hswidget
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/ianling/giu"
+	"github.com/ianling/imgui-go"
+
+	"github.com/OpenDiablo2/OpenDiablo2/d2common/d2enum"
+	"github.com/OpenDiablo2/OpenDiablo2/d2common/d2fileformats/d2cof"
+
+	"github.com/OpenDiablo2/HellSpawner/hscommon/hsenum"
+)
+
+const (
+	indicatorSize                        = 64
```

Comment:
oh, It was me. could you rename to `layerListW` or sth like that.

Reply:
resolved in 023fd2f